audio_effects.conf

```diff
@@ -12,6 +12,15 @@ libraries {
   reverb {
     path /system/lib/soundfx/libreverbwrapper.so
   }
+  qcbassboost {
```

Comment:
Explain the changes here. Were they done also on the 5.1 port? Why its needed etc. Try to be a bit more verbose, Its hard to approve things I dont know what they are doing.

Reply:
There are missing lib path to vendor/lib/soundfx on audio_effects.conf
And yes is also used on 5.1